modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/view_uad_entities.jsp

```diff
@@ -30,6 +30,8 @@ if (parentContainerId > 0) {
 
 	uadHierarchyDisplay.addPortletBreadcrumbEntries(request, renderResponse, locale);
 }
+
+long[] groupIds = (long[])request.getAttribute(UADWebKeys.GROUP_IDS);
```

Comment:
This should go before `ViewUADEntitiesDisplay viewUADEntitiesDisplay = (ViewUADEntitiesDisplay)request.getAttribute(UADWebKeys.VIEW_UAD_ENTITIES_DISPLAY);`